app/views/legislation/questions/show.html.erb

```diff
@@ -52,6 +52,11 @@
     </div>
 
     <aside class="small-12 medium-3 column">
+      <div id="supports" class="sidebar-divider"></div>
```

Comment:
```suggestion
      <div class="sidebar-divider"></div>
```

Most `sidebar-divider` elements in the application don't have an ID attribute, with the only exception being the `social-share` one.